src/main/javascript/src/views/participation/distribution/ParticipationCustomDistribution.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div>
+    <h1 class="mb-5">
+      Select the percent of students you would like to receive each condition
+    </h1>
+
+    <div class="row mx-2">
+      <div class="col-9 label">
+        Condition
+      </div>
+      <div class="col-3 label">
+        Distribution
+      </div>
+    </div>
+
+    <v-card class="mt-2 pt-5 px-5 mx-auto lighten-5 rounded-lg" outlined>
+      <v-card-text
+        v-for="(condition, index) in this.conditions"
+        :key="condition.conditionId"
+      >
+        <v-row class="mr-2">
+          <v-card-title class="col-10">
+            {{ condition.name }} will receive something
+          </v-card-title>
+          <v-text-field
+            class="textfield"
+            outlined
+            suffix="%"
+            v-model="distributionValue[index]"
+            :rules="[(value) => !!value || 'Required']"
+            required
+          ></v-text-field>
+        </v-row>
+      </v-card-text>
+    </v-card>
+
+    <p v-if="isDisabled()" class="errorMessage mt-3">
+      Please Provide Positive Value for Each Condition Distribution and All
+      Condition Distributions should be equal to 100%.
+    </p>
+
+    <v-btn
+      elevation="0"
+      class="mt-3"
+      :disabled="isDisabled()"
+      color="primary"
+      @click="updateDistribution()"
+      >Continue
+    </v-btn>
+  </div>
+</template>
+
+<script>
+import { mapActions } from "vuex";
+
+export default {
+  name: "ParticipationCustomDistribution",
+  props: ["experiment"],
+
+  data() {
+    return {
+      conditions: this.experiment.conditions,
```

Comment:
Does this work if you leave the step and come back? `this.experiment` is a prop so it may not always have a value yet when it's defined here. Consider using `computed` instead (https://vuejs.org/v2/guide/computed.html).

Reply:
Addressed the issue.